### ISSUE TITLE
Proposal for better gatekeeping: stick to masterpiece materials

### ISSUE THREAD

**Issue by guilsa** [tuvtran/project-based-learning#143]
### Description

Can we please refrain from adding things if content isn’t unique? 

### Why

I’ve been following the changes happening lately and I’m concerned making changes to this repo is turning into a popularity contest. Below are a few reasons why.

### Possible Implementations

I propose contributors ask themselves these questions before opening a PR:

- Does the writing follow a unique pedagogy? 
- Does the writing put you outside your comfort zone by taking an unfamiliar and scary topic and making you happy, instead of overwhelmed?
- Is the writing from a portal, aggregator or a commercial site (constains ads, etc)? That should be a yellow flag as these places favor breath over depth.

A few other random points: 

- I would prefer that topics ensure test of time and that they stay away from frameworks. Thoughts here?
- If we link to an aggregator site, we defeat the point of being curators here.

### Is this something you're interested in working on?

- [x] YES
- [ ] NO

**Comment by tuvtran** [comment]
I agree with your points. I'll admit that I haven't been able to pay enough attention to this project. I'll plan on purging some of the redundant tutorials. What do you think about putting them into a separate file like "Some other tutorials" which acts as a link aggregator, nothing more and nothing less?

**Comment by tuvtran** [comment]
we can begin tagging tutorials that stand the test of time like `classic`
